### DISCUSSION TITLE
Finalizing the spring demo election candidates and ballot questions

### DISCUSSION THREAD

**Discussion by windoverwater** [TrustTheVote-Project/VTP-web-api#14] [General]
It would be good to finalize the contests and ballot questions planned for the spring demo.

Since this is a smart phone demo based on a small screen, we would like to plan for and optimize for that.  Follow-on demos can be different.

The current constraints that we would like to agree on are:
- use the current shapes for party affiliation (circle, triangle, square, etc).  This will allow a simple glyph requiring little screen real-estate
- 25 character limit on names
- the presidential contest will be ticket based (president and vice-president)
- limit the number of contests to 5
- limit the number of candidates to 5
- have no more than two ballot questions

What follows are the current contests:

    "contests": {
        "GGOs/states/California": [
            {
                "US president": {
                    "choices": [
                        {
                            "name": "Mitt Romney",
                            "party": "Circle Party"
                        },
                        {
                            "name": "Phil Scott",
                            "party": "Circle Party"
                        },
                        {
                            "name": "Ron DeSantis",
                            "party": "Circle Party"
                        },
                        {
                            "name": "Kamala Harris",
                            "party": "Triangle Party"
                        },
                        {
                            "name": "Cory Booker",
                            "party": "Triangle Party"
                        },
                        {
                            "name": "Beta O'rourke",
                            "party": "Triangle Party"
                        }
                    ],
                    "tally": "rcv",
                    "uid": "0000"
                }
            },
            {
                "governor": {
                    "choices": [
                        {
                            "name": "Brian Kemp",
                            "party": "Circle Party"
                        },
                        {
                            "name": "Bernie Sanders",
                            "party": "Triangle Party"
                        }
                    ],
                    "tally": "plurality",
                    "uid": "0002"
                }
            }
        ],
        "GGOs/states/California/GGOs/counties/Alameda": [
            {
                "County Clerk": {
                    "choices": [
                        "Jean-Luc Picard",
                        "Huckleberry Finn",
                        "Peggy Carter"
                    ],
                    "tally": "rcv",
                    "uid": "0003"
                }
            }
        ],
        "GGOs/states/California/GGOs/towns/Alameda": [
            {
                "Question 1 - school budget override": {
                    "choices": [
                        true,
                        false
                    ],
                    "tally": "plurality",
                    "uid": "0006"
                }
            },
            {
                "Question 2 - new firehouse land purchase": {
                    "choices": [
                        true,
                        false
                    ],
                    "tally": "plurality",
                    "uid": "0007",
                    "win-by": "2/3"
                }
            }

**Comment by windoverwater** [comment]
Note - the first contest for president is important in that it is RCV and will be used to demo the insight that VTP can provide regarding RCV rounds.  FWIIW I wanted to also demonstrate how RCV tends towards centric candidates in that if people vote along actual political lines, Mitt Romney has a decent chance to become the winner as opposes to someone like perhaps Kamala Harris or Ron DeSantis.

On the other hand, there may be a compelling reason not to use actual political figures - TBD.

**Reply by stratofax** [reply]
Maybe @windoverwater we can just ping GM right now and ask him what he thinks about using real political figures?

**Comment by ttvp-admin** [comment]
And that is a grand question... we go over this all the time when
considering demos.  It is tough b/c the major problem is using real
(political) persons or contests can distract the use from experiencing the
feel of an app.  We tend to use characters that testers can get behind and
enjoy (George Jetson, Elroy Jetsaon, Astro Jetson, Fred, Wilma, Barney,
BamBam, Dino, etc.)  IF you are to use real candidates it can work
depending on your audience, but make sure they are perspectives and not
incumbents.  Forward-looking will engage the user in parsing the experience
as it triggers them to think what-if more easily and then engage them in
the process you're testing (in the case making choices on a ballot
screen).  Our design team (DDO) has done extensive focus-group testing in
this and it is fascinating how (in the world of politics and democracy
administration) the names and personas behind them can impact the testing
process.  Something to think about.  If it we're I, then I would carefully
consider a slate of candidates forward looking to the '24 election ( [D]
Joseph Biden/Kamala Harris, [R] Donald Trump/Nikki Haley, and [I] Bernie
Sanders/Elizabeth Warren) or simply [D] Jetsons and [R] Flintstones
characters

**Reply by stratofax** [reply]
As I discussed with @windoverwater yesterday, I think that we need to do everything we can do to keep the focus on the VTP system, and avoid the risk of distracting people with real politics.

In addition, and perhaps even more importantly, we're not implementing a secure system for the demo, partly because of time constraints, and partly because we're not planning on using VTP with anything else besides paper ballots, not phone-based voting systems.

But, if someone realizes they can "stuff the ballot box" by voting over and over for their favorite real candidate during the demo, the take-away becomes "we can't trust this voting system at all" instead of "this is the kind of innovation we need to save elections!"

On the other hand, if people are voting for, say, cartoon characters, they'll be constantly reminded that this is a demo, not a real election, and therefore focus instead on how the system works, instead of how the could use an online system to vote fraudulently for actual candidates.
